src/providers/discord.ts

```diff
@@ -163,27 +195,107 @@ const checkPermissions = async (channelId, botId) => {
 };
 
 client.on('ready', async () => {
-  ready = true;
   console.log(`[discord] bot logged as "${client.user.tag}"`);
   setActivity('!');
 
   await loadSubscriptions();
 });
 
+async function getEventsConfigured(guildId) {
+  const events = await db.queryAsync(
+    'SELECT events FROM subscriptions WHERE guild = ?',
+    guildId
+  );
+  return JSON.parse(events[0]?.events || `["proposal/start"]`);
```

Comment:
This field should never be empty, as:

- we specify when people use `/add` that they're subscribed to proposal/start
- we have a limit on the `/select-event` to always have at least one selection

We should remove the fallback value here, and update DB to change all empty `events` to `["proposal/start"]`